Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -298,6 +298,8 @@ void HexEditorWidget::update_inspector_values(size_t position)
         unsigned_64_bit_int = (unsigned_64_bit_int << (8 * bytes_left_to_read));
     }
 
+    Vector<u8> selected_bytes = m_editor->get_selected_bytes();
```

Comment:
Still, auto.

Reply:
I don't know what's going on, but this comment looks to be pointing to some old commit. GitHub is confused. Check the "files changed" tab and you'll see it's `auto`

Resolving this comment due to the misleading nature of it